ocpp-simulator/ocpp_simulator/cp_management/cp.py

```diff
@@ -0,0 +1,127 @@
+
+    async def send_start_transaction(self):
+        request = call.RequestStartTransactionPayload(
+            id_token={'idToken': str(uuid.uuid4()), 'type': 'Central'},
+            remote_start_id=1
+        )
+        response = await self.call(request)
+        return response
+
+    async def send_stop_transaction(self, transaction_id):
+        request = call.RequestStopTransactionPayload(
+            transaction_id=transaction_id
+        )
+        response = await self.call(request)
+        return response
+
+    async def send_authorize(self):
+        request = call.AuthorizePayload(
+            id_token={'idToken': str(uuid.uuid4()), "type": enums.IdTokenType.central},
+        )
+        response = await self.call(request)
+        return response
+
+    async def send_meter_value(self, transaction_id: int):
+        unit = datatypes.UnitOfMeasureType(
+            unit=enums.UnitOfMeasureType.w
+        )
+        request = call.MeterValuesPayload(
+            evse_id=1,
+            meter_value=[
+                datatypes.MeterValueType(
+                    timestamp=str(datetime.now()),
+                    sampled_value=[
+                        datatypes.SampledValueType(
+                            value=10,
+                            context='Sample.Periodic',
+                            measurand='Power.Active.Import',
+                            phase=None,
+                            location=None,
+                            unit_of_measure=unit
+                        ),
+                        datatypes.SampledValueType(
+                            value=10,
+                            context='Sample.Periodic',
+                            measurand='Power.Active.Import',
+                            phase='L1',
+                            location=None,
+                            unit_of_measure=unit
+                        ),
+                    ]
+                ),
+                datatypes.MeterValueType(
+                    timestamp=str(datetime.now() + timedelta(seconds=5)),
+                    sampled_value=[
+                        datatypes.SampledValueType(
+                            value=10,
+                            context='Sample.Periodic',
+                            measurand='Power.Active.Import',
+                            phase=None,
+                            location=None,
+                            unit_of_measure=unit
+                        ),
+                        datatypes.SampledValueType(
+                            value=10,
+                            context='Sample.Periodic',
+                            measurand='Power.Active.Import',
+                            phase='L1',
+                            location=None,
+                            unit_of_measure=unit
+                        ),
+                    ]
+                ),
+            ]
+        )
+        response = await self.call(request)
+        return response
```

Comment:
what about receiving messages?
no all the messages are implemented.
payloads must be input from user